tools/CustomTasks/DependencyPropertyCodeGen.cs

```diff
@@ -773,13 +773,16 @@ private string WriteImplementation(TypeDefinition typeDefinition, List<TypeDefin
 @"void {0}Properties::{1}({2} {3}value)
 {{", ownerType.Name, prop.Name, prop.PropertyCppName, CppInputModifier(prop.PropertyType)));
                     string localName = "value";
+                    sb.AppendLine(@"    [[gsl::suppress(con)]]
```

Comment:
Out of curiosity, what does this?

Reply:
I'm not sure what the question is, This changes the code gen to add this line to the code gen files which informs the compiler to ignore the con gsl rule within this block. We need this here because some of the coerce methods modify the value and some do not meaning const and non-const are neither correct in all cases so the rule is not enforceable without changing the coerce functions.

Reply:
Oh right I see, thanks for the explanation!